index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html style="background: #0f0f0f" lang="en">
+<html lang="en">
```

Comment:
When initially loading the page now, the background is white until it got loaded which is actually what got avoided by setting the background here. Why did you remove it?

Reply:
> When initially loading the page now, the background is white until it got loaded which is actually what got avoided by setting the background here. Why did you remove it can't we just change it dynamically or  just remove its padding so that the children fill the whole size?

Reply:
I didn't consider that, sorry!